systemtest/insights-core-setup.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/bash -ux
+
+# Stolen from
+# https://github.com/RedHatInsights/insights-core/commit/8560f30fef86548703f408c76ac0685085c2ca37#diff-b96e297c07a75e043af16f92339e07d3bb8665e234dc37809be299c1513c36dbR9-R18
+# TODO: find a nicer link once that is merged ^
```

Comment:
It has been merged: https://github.com/RedHatInsights/insights-core/blob/master/insights/tests/client/test_crypto.py